README.md

```diff
@@ -48,11 +48,49 @@ OpenGL wrappers:
 Vulkan wrappers:
 
 - `auto.nixVulkanNvidia`: Proprietary Nvidia driver (auto detection).
-- `auto.nixVulkanIntel`: Mesa Vulkan implementation (auto detection).
+- `nixVulkanIntel`: Mesa Vulkan implementation.
 
 The Vulkan wrapper also sets `VK_LAYER_PATH` the validation layers in the nix store.
 
-## flakes
+## Flakes
+
+### Directly run nixGL
+
+```sh
+nix run --impure github:guibou/nixGL -- program
+```
+You can also specify which wrapper to use instead of using the default auto detection:
+
+```sh
+nix run github:guibou/nixGL#nixGLIntel -- program
+```
+
+This will result in a lighter download and execution time. Also, this evaluation is pure.
+
+#### Error with GLIBC version
+
+if you get errors with messages similar to
+```
+/nix/store/g02b1lpbddhymmcjb923kf0l7s9nww58-glibc-2.33-123/lib/libc.so.6: version `GLIBC_2.34' not found (required by /nix/store/hrl51nkr7dszlwcs29wmyxq0jsqlaszn-libglvnd-1.4.0/lib/libGLX.so.0)
+```
+
+You may need to run `nixGL` based on an older `nixpkgs` (defaults to `nixos-21.11`) like so:
+
+```sh
+nix run --override-input nixpkgs nixpkgs/nixos-21.05 --impure github:guibou/nixGL -- program
+```
+
```

Comment:
Could you add a sentence which emphasis that this is usually a requierement. Something such as

> As a rule of thumb, use the same nixpkgs clone for `nixGL` than the one used for your `program`